cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/OptionalLoginCommandOptions.java

```diff
@@ -0,0 +1,73 @@
+package cloud.foundry.cli.services;
+
+import picocli.CommandLine.Option;
+import picocli.CommandLine.ScopeType;
+
+/**
+ * Common options for the initialization process of the
+ * {@link org.cloudfoundry.operations.CloudFoundryOperations operations} object.
+ *
+ * @see cloud.foundry.cli.crosscutting.mapping.CfOperationsCreator
+ */
```

Comment:
The documentation should mention that the user is not forced to provide target information for these `LoginCommandOptions`